test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/BuildIntegrationTest.cs

```diff
@@ -614,6 +614,22 @@ public async Task Build_WithDeterministicFlagSet_OutputsDeterministicViewsAssemb
             Assert.Equal(firstAssemblyBytes, secondAssemblyBytes);
         }
 
+        [Fact]
+        [InitializeTestProject("SimpleMvc21")]
+        public async Task Building_NETCoreApp21TargetingProject()
```

Comment:
Verifying that 2.1 projects build when using the new Sdk. Will add a scenario for 2.2 once it ships

Reply:
Log an issue :)

Reply:
https://github.com/aspnet/Razor/issues/2677